CNV storage updates (#64685)
openshift/openshift-docs: virt/storage/virt-automatic-bootsource-updates.adoc

:_content-type: ASSEMBLY
[id="virt-automatic-bootsource-updates"]
= Managing automatic boot source updates
include::_attributes/common-attributes.adoc[]
:context: virt-automatic-bootsource-updates

> 

Author: An oversight! I will fix that.

:_content-type: ASSEMBLY
[id="virt-automatic-bootsource-updates"]
= Managing automatic boot source updates
include::_attributes/common-attributes.adoc[]
:context: virt-automatic-bootsource-updates

toc::[]

You can manage automatic updates for the following boot sources:

* xref:../../virt/storage/virt-automatic-bootsource-updates.adoc#managing-rh-boot-source-updates_virt-automatic-bootsource-updates[All Red Hat boot sources].
* xref:../../virt/storage/virt-automatic-bootsource-updates.adoc#managing-custom-boot-source-updates_virt-automatic-bootsource-updates[All custom boot sources].

> * xref:../../virt/storage/virt-automatic-bootsource-updates.adoc#virt-disable-auto-updates-single-boot-source_virt-automatic-bootsource-updates[Individual Red Hat or custom boot sources].

Reviewer: No periods needed for these list items.

> Boot sources can make virtual machine (VM) creation more accessible and efficient for users. If automatic boot source updates are enabled, the Containerized Data Importer (CDI) imports, polls, and updates the images so that they are ready to be cloned for new VMs. By default, CDI automatically updates Red Hat boot sources.

Reviewer: ```suggestion
Boot sources can make virtual machine (VM) creation more accessible and efficient for users. If automatic boot source updates are enabled, the Containerized Data Importer (CDI) imports, polls, and updates the images so that they are ready to be cloned for new VMs. By default, the CDI automatically updates Red Hat boot sources.
```

[id="managing-rh-boot-source-updates_virt-automatic-bootsource-updates"]
== Managing Red Hat boot source updates

> You can opt out of automatic updates for all system-defined boot sources by disabling the `enableCommonBootImageImport` feature gate. If you disable this feature gate, all `DataImportCron` objects are deleted. This does not remove previously imported `PersistentVolumeClaim` (PVC) objects that store operating system images, though administrators can delete them manually.

Reviewer: ```suggestion
You can opt out of automatic updates for all system-defined boot sources by disabling the `enableCommonBootImageImport` feature gate. If you disable this feature gate, all `DataImportCron` objects are deleted. This does not remove previously imported `PersistentVolumeClaim` (PVC) objects that store operating system images, which administrators can delete manually.
```
